README.md

```diff
@@ -44,82 +43,54 @@ The middleware is a caching and reporting layer which sits in front of the nodes
 
 The architecture of the app is explained [here](docs/architecture.md).
 
-## Prerequisites
-
-Ensure that you have [Elixir](https://elixir-lang.org/install.html) installed, using Erlang 22 or newer.
 
 ## Setup
 
 Firstly, clone the middleware repo:
 
 `git clone https://github.com/aeternity/ae_mdw && cd ae_mdw`
 
-For syncing on testnet, change ./docker/aeternity.yaml network_id to `ae_uat`.
-
-Following there are two options depending on which block height you want to continue syncing:
-
-### A) Sync from genesis block
+Before running it, it's recommended to use a Node database snapshot for faster syncing.
 
-1. Simply run the middleware with `docker-compose up`
+### Database Snapshot
 
-Next time you run the same command it will continue from the last height synced.
+1. Download one of the full backups from https://downloads.aeternity.io
```

Comment:
does this work, I assume they are 6.7.0 based and we are still running 6.5.2?

Reply:
It works for first and last testnet ones.

Reply:
cool, good to know